TSC-1/README.md

```diff
@@ -1,128 +1,7 @@
-# ODK TSC Landing Page
+# TSC 1
 
-The Technical Steering Committee is the technical governing body for Open Data Kit 1 core projects.
+TSC 1 is responsible for ODK Collect, ODK Aggregate, ODK XLSForm, ODK Build, ODK Briefcase, ODK Central, ODK JavaRosa, and the ODK XForms specification.
```

Comment:
Is this going to be the authoritative list of these? I hope so!

Reply:
It has to be other places (e.g., http://opendatakit.org/community/governance/), but this can be the authoritative list.

Reply:
But it's also in the main TSC document... Can we convert this to just refer to that?

Reply:
Yeah, I see that now. Working on it...